code/modules/surgery/organs/stomach.dm

```diff
@@ -48,6 +49,17 @@
 		if(DISGUST_LEVEL_DISGUSTED to INFINITY)
 			H.throw_alert("disgust", /obj/screen/alert/disgusted)
 
+/obj/item/organ/stomach/proc/handle_nutrition(mob/living/carbon/human/H)
+	//First, calculate the actual vitamin levels
+	var/vitamin_decay_rate = H.vitamins * 0.001 //Vitamin levels will slowly decay towards neutral
+	vitamin_decay_rate = Clamp(vitamin_decay_rate, -0.05, 0.05)
+	H.vitamins -= vitamin_decay_rate //Yes, this works for negatives, don't ask me how

```

Comment:
it works for negatives because if h.vitamins is negative then VDR is negative and subtracting a negative pushes upward

Reply:
I spent at *least* an hour trying to figure that out. Ugh.